notebooks/02_analyses/sentiment_analysis.ipynb

```diff
@@ -0,0 +1,531 @@
+{
```

Comment:
<p>please move <code>load_dotenv("../../.env")</code>  to the bottom of this cell</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/mailing-list-analysis-toolkit/pull/66/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>